spec/models/user_spec.rb

```diff
@@ -13,7 +13,12 @@
     it {is_expected.to have_db_column :email}
     it {is_expected.to have_db_column :encrypted_password}
   end
-  
+
+  describe 'db table with added fb-login' do
+    it {is_expected.to have_db_column :uid}
+    it {is_expected.to have_db_column :provider}
+  end
```

Comment:
You don't need a separate describe block for these tests. I would add them to the existing describe block for the database columns.

Reply:
Tnx! Will change this 👍